src/tabs/options.html

```diff
@@ -88,6 +88,14 @@
                     </div>
                     <span class="freelabel" i18n="showNotifications"></span>
                 </div>
+                <div class="backupOnFlash margin-bottom">
+                    <select id="backupOnFlashSelect">
+                        <option value="0" i18n="firmwareBackupDisabled"></option>
+                        <option value="1" i18n="firmwareBackupEnabled"></option>
+                        <option value="2" i18n="firmwareBackupAsk"></option>
```

Comment:
when i clear my cache, and refresh the page. disabled (0) shows as default.  but the code states otherwise. 
please test with data cleared.
![image](https://github.com/user-attachments/assets/b5685448-9dcf-4e2d-aa2d-3fdc3eaeda65)